src/main/java/com/gmail/luchyk/viktoriia/glovodb/dto/Address.java

```diff
@@ -0,0 +1,12 @@
+package com.gmail.luchyk.viktoriia.glovodb.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Address {
```

Comment:
If you change name the class AddressDto, you can rename the method toAddress in AddressConverter to toDto. I think it will be more logical - AddressConverter.toDto(). The same with anothers Dtos